internal/db/src/schema/identity.ts

```diff
@@ -9,20 +9,21 @@ export const identities = mysqlTable(
   {
     id: varchar("id", { length: 256 }).primaryKey(),
     /**
-     * The extenral id is used to create a reference to the user's existing data.
+     * The external id is used to create a reference to the user's existing data.
      * They likely have an organization or user id at hand
      */
     externalId: varchar("external_id", { length: 256 }).notNull(),
     workspaceId: varchar("workspace_id", { length: 256 }).notNull(),
     environment: varchar("environment", { length: 256 }).notNull().default("default"),
-    ...lifecycleDates,
     meta: json("meta").$type<Record<string, unknown>>(),
+    deleted: boolean("deleted").default(false),
```

Comment:
this should have a `notNull()`, otherwise it's quite annoying having to deal with the `NullBool` in go

Reply:
Yeah agreed, i still have to do the actual changes for soft/hard deleting will fix it with that commit